src/commands/instance/list.ts

```diff
@@ -0,0 +1,24 @@
+import cli from 'cli-ux'
+import {Instance} from 'mesg-js/lib/api'
+
+import Command from '../../root-command'
+
+export default class InstanceList extends Command {
+  static description = 'List all instances'
+
+  static flags = {
+    ...Command.flags,
+    ...cli.table.flags()
+  }
+
+  async run(): Promise<Instance[]> {
+    const {flags} = this.parse(InstanceList)
+    const {instances} = await this.api.instance.list({})
+    if (!instances) return []
+    cli.table(instances, {
+      hash: {header: 'HASH', get: x => x.hash},
+      sid: {header: 'SERVICE', get: x => x.serviceHash},
+    }, {printLine: this.log, ...flags})
```

Comment:
Displaying the name of the service is useful. Otherwise we only see hashes:
```
➜  cli git:(feature/api-integration) ./bin/run instance:list
HASH                                         SERVICE                                      
J7ot3GBb3y4mWMwRqwSMnsFkui27Nen13Gr97N7DLVbK 2BypXmch8gNTvi9wgketkCnBnCZkEetwp3TjnvvVTEAb 
```

Reply:
done